src/pal/src/configure.cmake

```diff
@@ -130,6 +130,8 @@ check_function_exists(pthread_mutex_init HAS_PTHREAD_MUTEXES)
 check_function_exists(ttrace HAVE_TTRACE)
 check_function_exists(pipe2 HAVE_PIPE2)
 
+check_symbol_exists(getrandom sys/random.h HAVE_GETRANDOM)
```

Comment:
We build our official builds on CentOS 7 to allow them to work on a large number of distros. CentOS 7 headers don't contain getrandom definition, although they do have __NR_getrandom defined. So we could use syscall function with that instead.

Reply:
Ah, right. Yes, good suggestion. Will do this.